packages/react-storage/src/components/StorageBrowser/actions/configs/types.ts

```diff
@@ -80,63 +84,93 @@ export interface TaskActionConfig<T extends TaskHandler> {
   /**
    * whether the action allows inflight cancellation
    */
-  isCancelable: IsCancelableTaskHandler<T>;
+  isCancelable: boolean;
```

Comment:
This change breaks restricting `isCancelable` based on the allowed `result` of the  provided `handler`. 

For example, if i provide a `handler` with a `CancelableTaskHandlerOutput`, the value of `result` is :
```
"CANCELED" | "COMPLETE" | "FAILED"
```
`IsCancelableTaskHandler` will restrict `isCancelable` to `true`. For context, `isCancelable` is ultimately a runtime flag used to determine whether or not to display the "Canceled" count in the `StatusDisplay`

Reply:
This restriction makes `isCancelableTaskHandler` field un-configurable while defining a custom action, where the `result` is not a part of the custom action interface to infer the correct value from. (with the restriction, `isCancelableTaskHandler` value will always be `false` within custom actions) Do we need an interface change for this?

Reply:
But also open to interface changes if need be 😄